src/config.rs

```diff
@@ -44,6 +46,30 @@ pub struct ExportersConfig {
     pub influxdb: InfluxDBConfig,
 }
 
+#[derive(Debug, Deserialize, PartialEq)]
+#[serde(rename_all = "lowercase")]
+#[serde(remote = "LevelFilter")]
+pub enum LevelFilterDef {
+    Off,
+    Error,
+    Warn,
+    Info,
+    Debug,
+    Trace,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct LoggerConfig {
+    #[serde(with = "LevelFilterDef")]
+    #[serde(default = "default_log_level_filter")]
+    pub level_filter: LevelFilter,
```

Comment:
I think better name would be `log_level` for this field.

Reply:
I named it this way because we have `LoggerConfig` already in the namespace, so we could avoid the `log_` prefix and as the type is called `LevelFilter`, it feels natural to read `LoggerConfig.level_filter`. What do you think?

Reply:
In understand that inside our application `level_filter` makes more sense but in the configuration file `log_level` (or just `level` since it is under "logging"). `log_level` feels more natural to me in a configuration file. If you wish to use different name internally, maybe you can use serde's rename attribute on the field?